<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1011</width>
    <height>651</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>0</y>
      <width>921</width>
      <height>431</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <pointsize>13</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Программа для работы с АВЛ деревом</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QTableWidget" name="tableWidget">
           <property name="font">
            <font>
             <pointsize>11</pointsize>
            </font>
           </property>
           <column>
            <property name="text">
             <string>Ключ</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Информация</string>
            </property>
           </column>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <layout class="QGridLayout" name="gridLayout_3">
             <item row="3" column="2">
              <widget class="QComboBox" name="comboBox">
               <property name="font">
                <font>
                 <pointsize>11</pointsize>
                </font>
               </property>
               <item>
                <property name="text">
                 <string>В прямом порядке</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>В обратном порядке</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>По возрастанию ключа</string>
                </property>
               </item>
              </widget>
             </item>
             <item row="4" column="1">
              <widget class="QPushButton" name="AddButton">
               <property name="font">
                <font>
                 <pointsize>11</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Добавить запись</string>
               </property>
              </widget>
             </item>
             <item row="1" column="2">
              <widget class="QLineEdit" name="FindEdit">
               <property name="font">
                <font>
                 <pointsize>11</pointsize>
                </font>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="maxLength">
                <number>9</number>
               </property>
               <property name="placeholderText">
                <string>Ключ</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QPushButton" name="DeleteButton">
               <property name="font">
                <font>
                 <pointsize>11</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Удалить по ключу</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QPushButton" name="FindButton">
               <property name="font">
                <font>
                 <pointsize>11</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Найти по ключу</string>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="QPushButton" name="PrintButton">
               <property name="font">
                <font>
                 <pointsize>11</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Распечатать информацию</string>
               </property>
              </widget>
             </item>
             <item row="6" column="1" colspan="2">
              <widget class="QPushButton" name="VisualizationButton">
               <property name="font">
                <font>
                 <pointsize>11</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Визуализация дерева</string>
               </property>
              </widget>
             </item>
             <item row="4" column="2">
              <layout class="QGridLayout" name="gridLayout_2">
               <item row="0" column="1">
                <widget class="QLineEdit" name="Add_KeyEdit">
                 <property name="font">
                  <font>
                   <pointsize>11</pointsize>
                  </font>
                 </property>
                 <property name="styleSheet">
                  <string notr="true"/>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                 <property name="maxLength">
                  <number>9</number>
                 </property>
                 <property name="placeholderText">
                  <string extracomment="Ауе">Ключ</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QLineEdit" name="Add_InfoEdit">
                 <property name="font">
                  <font>
                   <pointsize>11</pointsize>
                  </font>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                 <property name="maxLength">
                  <number>15</number>
                 </property>
                 <property name="placeholderText">
                  <string extracomment="Ауе">Информация</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item row="2" column="2">
              <widget class="QLineEdit" name="DeleteEdit">
               <property name="font">
                <font>
                 <pointsize>11</pointsize>
                </font>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="maxLength">
                <number>9</number>
               </property>
               <property name="placeholderText">
                <string>Ключ</string>
               </property>
              </widget>
             </item>
             <item row="5" column="1" colspan="2">
              <widget class="QPushButton" name="NumOfNodesButton">
               <property name="font">
                <font>
                 <pointsize>11</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Число узлов с указателями на одну ветвь</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>72</x>
      <y>440</y>
      <width>921</width>
      <height>161</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QTableWidget" name="arrayWidget">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <row>
        <property name="text">
         <string>Ключ</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>Информация</string>
        </property>
       </row>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QPushButton" name="ClearButton">
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Очистить массив</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="AddCellButton">
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Добавить ячейку в массив</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="InputButton">
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Ввести данные из массива</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1011</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
